test/dynamics/maximum_height_from_velocity_test.py

```diff
@@ -0,0 +1,36 @@
+from collections import namedtuple
+from pytest import approx, fixture, raises
+from symplyphysics import (
+    errors,
+    units,
+    convert_to,
+    Quantity,
+    SI,
+)
+from symplyphysics.laws.dynamics import maximum_height_from_velocity as maximum_height_law
+
+# Description
+## If a body is thrown vertically upwards at a speed of 10 m/s, it will rise to a height of  (10*10) /2*9,8 = 5.1 meters
+
+@fixture(name="test_args")
+def test_args_fixture():
+    v = Quantity(10 * units.meter / units.second)
+    Args = namedtuple("Args", ["v"])
+    return Args(v=v)
+
+
+def test_basic_height(test_args):
+    result = maximum_height_law.calculate_height(test_args.v)
```

Comment:
It does not work. Check your code before submission.

Reply:
changed function name